theme/assets/css/src/editor.css

```diff
@@ -57,3 +64,36 @@ blockquote {
 .top-app-bar__title a {
 	color: var(--mdc-theme-on-header, var(--mdc-theme-on-primary, #fff));
 }
+
+.tab-bar__wrap {
+	width: 100%;
+}
+
+/* Navigation */
+
+.wp-block-material-navigation .wp-block-navigation-placeholder__preview {
+	display: flex;
+	width: 100%;
+}
+
+.wp-block-material-navigation.is-selected .wp-block-navigation-placeholder__controls {
+	display: flex;
+	margin-top: -48px;
+}
+
+.wp-block-material-navigation.is-selected .wp-block-navigation-placeholder__preview:not(.is-loading) {
+	display: flex;
+	flex: 0;
+	flex-wrap: nowrap;
+	opacity: 0;
+	overflow: hidden;
+	width: 0;
+}
+
+.wp-block-site-logo {
+
+	& .components-resizable-box__container {
+		height: auto !important;
+		width: auto !important;
+	}
+}
```

Comment:
Somehow `mdc-theme-header` is not injecting, and material style is using primary color for header.